deploy/data_exporter/data_export_function/app.py

```diff
@@ -0,0 +1,178 @@
+import os
+import subprocess
+from datetime import datetime, timedelta, timezone
+
+import boto3
+import psycopg
+from psycopg import sql
+
+ssm = boto3.client("ssm")
+s3 = boto3.client("s3", region_name="eu-west-1")
+bucket_name = "dc-ynr-short-term-backups"
+current_time = datetime.now().isoformat()
+PREFIX = "ynr-export"
+FILENAME = f"{PREFIX}-{current_time.replace(':', '-')}.dump"
```

Comment:
I know lambdas are short lived and everything but setting `current_time = datetime.now().isoformat()` at module load time is making me feel itchy. I'd probably do

```py
def get_filename():
    current_time = datetime.now().isoformat()
    return f"{PREFIX}-{current_time.replace(':', '-')}.dump"
```
and then call `get_filename()` once at the start of `dump_and_export()`. That just feels safer to me.

Reply:
https://github.com/DemocracyClub/yournextrepresentative/pull/2469/commits/577723cd64dc414a4d0a52e1488480b33b91796c